README.md

```diff
@@ -41,12 +43,110 @@ This plugin can be consumed by the CAP application deployed on BTP to store thei
 >
 > A reference to adding this can be found [here](https://github.com/cap-java/sdm/blob/691c329f4c3c17ae390cfcb2db1ef02650585aee/cap-notebook/demoapp/pom.xml#L20)
 
-## Deploying and testing the application
+## Setup
 
-1. Log in to Cloud Foundry space:
+In this guide, we use the Bookshop sample app in the [deploy branch](https://github.com/cap-java/sdm/tree/deploy) of this repository, to integrate SDM CAP plugin.  Follow the steps in this section for a quick way to deploy and test the plugin without needing to create your own custom CAP application.
 
-   ```sh
+### Using the released version
+If you want to use the version of SDM CAP plugin released on the central maven repository follow the below steps:
+
+1. Remove the sdm and sdm-root folders from your local .m2 repository. This ensures that the CAP application uses the plugin version from the central Maven repository, as the local .m2 repository is prioritized during the build process.
+
+2. Clone the sdm repository:
+
+```sh
+   git clone https://github.com/cap-java/sdm
+```
+
+3. Checkout to the branch **deploy**:
+
+```sh
+   git checkout deploy
+```
+
+4. Navigate to the demoapp folder:
+
+```sh
+   cd cap-notebook/demoapp
+```
+
+5. Configure the [REPOSITORY_ID](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/mta.yaml#L21) with the repository you want to use for deploying the application. Set the SDM instance name to match the SAP Document Management integration option instance you created in BTP. Update these details in the mta.yaml file under the [srv module](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/mta.yaml#L31) and the [resources section](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/mta.yaml#L98) values in the **mta.yaml**. 
+
+6. Build the application:
+
+```sh
+   mbt build
+```
+Now the application will pick the released version of the plugin from the central maven repository as the dependency is added in the [pom.xml](https://github.com/cap-java/sdm/blob/4180e501ecd792770174aa4972b06aff54ac139d/cap-notebook/demoapp/srv/pom.xml#L18)
+
+7. Log in to Cloud Foundry space:
+
+```sh
    cf login -a <CF-API> -o <ORG-NAME> -s <SPACE-NAME>
+```
+8. Deploy the application:
+
+```sh
+   cf deploy mta_archives/*.mtar
+```
+
+### Using the development version
+To use a development version of the SDM CAP plugin, follow these steps. This is useful if you want to test changes made in a separate branch or use a version not yet released on the central Maven repository.
```

Comment:
"if you want to test changes made in a separate branch of this github repository"

Reply:
Done